setup.bash

```diff
@@ -20,7 +20,9 @@ then
     ./configure --enable-optimizations
     make -j4
     sudo make altinstall
+    
     echo "alias python=/usr/local/bin/python3.7" >> ~/.bashrc
```

Comment:
Shouldn't you use the `version` variable you define in line 14 inside the echo? Otherwise changes in this variable won't have effect here.

Reply:
the folder is named python3.7 and not python3.7.3, and we use the variable version to get the binary file from source

Reply:
Yes, but if you change `version` file in line 18 (line 14 in this PR), for example to 3.8.X, you would be installing that other  version but your alias (and path's python) would still be python3.7 that doesn't correspond to what you downloaded? Because in alias you are giving a binary, not a folder

Reply:
The alias point to the folder which python where installed, and no to the binary  downloaded, if we change the version for 3.8.X for example we need to change the alias to point to folder where the new version(3.8) was been installed

Reply:
I see. I think it's strange you download a Python-$version.tgz, but then regardless of what you download, you still go to python3.7...
I think it's very easy to have an unwanted effect here: someone changes $VERSION and then doesn't change the rest. Why not create a $VERSION_SHORT so the alias and the path are always used for the version you install?

All I'm saying is, given you have a general variable to select the python you download, I think you would have it complete by also specifying the alias/path to the version you install automatically.